### PR TITLE
added a new Typedef for Bans you can get from <Guild>.fetchBans() and fixed a little typo in the fetchAuditLogs() method

### PR DESCRIPTION
**Pull request by Dev-Yukine**
**Please describe the changes this PR makes and why it should be merged:**
added a new typedef for Discord Bans because it looks cleaner than write it in the description and fixed a little typo in fetchAuditLogs.type type so its now correct and clickable

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/Guild.js [hunk 1]
```diff
@@ -432,9 +432,15 @@ class Guild extends Base {
   }
 
   /**
+ * A Discord Guild Ban.
```
Comment (devsnek) [inline src/structures/Guild.js]: `Discord Ban Object`

Comment (Dev-Yukine) [reply src/structures/Guild.js]: ok 👀

**Comment by bdistin** [comment]
Can you also fix the indentation?

**Comment by Dev-Yukine** [comment]
oh do you mean the blank line between the typedef and the method docs? @bdistin

**Comment by bdistin** [comment]
img top docstring needs more indentation for the body, the bottom docstring shows correct indentation.

**Comment by bdistin** [comment]
👍

File: src/structures/Guild.js [hunk 2]
```diff
@@ -432,9 +432,15 @@ class Guild extends Base {
   }
 
   /**
+   * A Discord Ban Object.
+   * @typedef {Object} Ban
```
Comment (KingDGrizzle) [inline src/structures/Guild.js]: I'd call this `BanObject`, not just `Ban`, as it could be confused with other things. But thats just me

Comment (Dev-Yukine) [reply src/structures/Guild.js]: i mean the description tells that tbh and fetchBans is the only method that is using that type so idk but you're maybe right

**Pull request merged**